prob-1/prob1.py

```diff
@@ -0,0 +1,7 @@
+sum = 0;
+
+for i in range (1000):
+    if (i % 3 == 0 or i % 5 == 0):
+        sum += i;
```

Comment:
This for loop is written better in functional format: First collect the items you want using a comprehension, then sum them using the `sum` function.